issue_36.md

```diff
@@ -0,0 +1,32 @@
+# Segment recordings by runs
+
+#36
+Ideas for name : split4phys, split4runs, obj2split,  split4bids, rec2run, phys2runs, ses2run, 4run2split
+
+# The option we settled for :
+Integrate the splitting function with phys2bids.
+
+# What we need to do
+## 1.Adapt phys2bids parser arguments, in `run.py`
+- [ ] ntp type, from int to list
+- [ ] tr type, from float to list
+- [ ] (uncertain) either add argument to specify the file is multi-run, or detect if list contains multiple item.
+
+```
+if multiple item (in ntp or in tr):
+  -run is True.
+
+If run is True:
+  redirect to part of code that splits the file
+```
+
+## 2.Ascertain lists length before running main workflow, after loading file (in `phys2bids.py, line 241`)
+- [ ] Raise appropriate errors (e.g. ntp list has 1 item but `-run` is True, ntp has more than 1 item and tr has more) - in `split2phys.py line 75 to 90`
+- [ ] add padding to non-equivalent list (in `split2phys.py line 92`)
+## 3.Check that sum(ntp_list) gives the right amount of trigger tps
+- [ ] adapt check_trigger_amount section in `phys2bids.py, line 265`, with `split2phys.py, line 118`
+- [ ] find the right value to give to `tr` parameter when running `phys_in.check_trigger_amount()` (no hard-coded value)
```

Comment:
Don't worry about this: TR = 1 when you run the first, general, check_trigger_amount() is more than ok. Think it this way: the only reason for which we use the TR information NOW is to know how much to anticipate the first TR, in case any is missing. But when we deal with the first, general, multi-run timepoints check, it might be a much better call to just break the program if we can't find the same number of timepoints as the user provided.
The other thing for which we'll use the info about TR will be to know when a run ends, so that we can cut the recordings. However, that use is based on the idea that your TR is always the same. When you do the general check for the triggers in a multi-run file you might have different TRs in different runs, but most of all you'll have the time between recordings that most probably won't be the same as the TR. Using a "real" TR becomes complicated in that case, as you don't know it any more. Besides, you don't need to know it because you're not cutting anything yet.